Includes/RMCLogger.hpp

```diff
@@ -36,7 +36,7 @@ namespace CTRPluginFramework {
                 u8 userPacketNote : 1;
             } flags;
         };
-        
+        u64 titleID;
```

Comment:
This can be placed inside `PacketMetadata`, which saves a lof of size calculations. Make sure to set `PacketMetadata` as `PACKED`

Reply:
I've made this change and pushed it, but for some reason the `isRecievedPacked` flag is encoded differently now? It's now gets encoded as `0x60` if false, and `0x61` if true
![hokakuctr-before](https://github.com/PretendoNetwork/HokakuCTR/assets/27011796/a38f196f-469c-476a-8ce6-053126fca0de)
![hokakuctr-after](https://github.com/PretendoNetwork/HokakuCTR/assets/27011796/4e5f2bcf-d685-41bd-a447-a47a3b63e26d)

Reply:
Because you probably put it inside the flags and not below it.

Reply:
Looks like he put it above the flags though

Reply:
> Because you probably put it inside the flags and not below it.

`flags` remains unchanged
![Screenshot from 2023-10-14 12-19-55](https://github.com/PretendoNetwork/HokakuCTR/assets/27011796/38134565-c82e-47fc-8f6d-8e5b0db7008c)

Reply:
@PabloMK7 @DaniElectra I'm unsure why this happens but it does seem to be caused by the `flags` struct. Since `userPacketNote` was unused here I removed it, and then since there was only the one flag left I just removed the `flags` struct altogether and placed `isRecievedPacked` inside `PacketMetadata` right under `titleID`. Doing this made it encode to 0 and 1 again, like before

Is this acceptable?

Reply:
Sure, that's fine to me.

Reply:
> Wait actually, try this
> 
> ```
> struct {
> u8 isRecievedPacket : 1;
> u8 userPacketNote : 1;
> u8 unused : 6;
> } flags;
> ```

This still encodes it as `0x60` and `0x61`. I'm gonna push what I have without the `flags` struct